lib/module/stories/stories_view.dart

```diff
@@ -136,6 +144,14 @@ class HnStoriesPageState extends State<HnStoriesPage>
               _handleThemeChange(ThemeName.dark);
             },
           ),
+          const Divider(),
+          new ListTile(
+            title: const Text('Show full comments'),
+            trailing: new Checkbox(
```

Comment:
Yep, originally I wanted to use [SwitchListTile](https://docs.flutter.io/flutter/material/SwitchListTile-class.html) but it seeems it is no longer available. For some reason forgot about Switch though...